src/geode/geometry/distance.cpp

```diff
@@ -70,6 +70,7 @@ namespace
     {
         const auto p =
             ( point_to_v0_length + point_to_v1_length + segment_length ) / 2;
+        if( p - point_to_v0_length <= geode::global_epsilon )
```

Comment:
This is a WTF bug!!!

Reply:
When had this line been removed? If the bug was not introduced when I pushed the code... :o